var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -108,15 +103,26 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     # ^ need to depend when both hpctoolkit and tau
 
     # HiOp dependency logic
-    depends_on("hiop+raja", when="+hiop+raja")
     depends_on("hiop@0.3.99:", when="@0.99:+hiop")
     depends_on("hiop@0.5.1:", when="@1.1.0:+hiop")
     depends_on("hiop@0.5.3:", when="@1.3.0:+hiop")
-    depends_on("hiop@0.7.0:0.7.1", when="@1.5.0:+hiop")
+    depends_on("hiop@0.7.0:", when="@1.5.0:+hiop")
 
     depends_on("hiop~mpi", when="+hiop~mpi")
     depends_on("hiop+mpi", when="+hiop+mpi")
 
+    # RAJA dependency logic
+    # ExaGO will support +raja~hiop in the future
+    depends_on("hiop+raja", when="+hiop+raja")
+    # This is duplicated from HiOp
+    # RAJA > 0.14 and Umpire > 6.0 require c++ std 14
+    # We are working on supporting newer Umpire/RAJA versions
+    depends_on("raja@0.14.0:0.14", when="@1.1.0:+raja")
+    depends_on("umpire@6.0.0:6", when="@1.1.0:+raja")
+    depends_on("camp@0.2.3:0.2", when="@1.1.0:+raja")
+    # This is no longer a requirement in RAJA > 0.14
+    depends_on("umpire+cuda~shared", when="+raja+cuda")
```

Comment:
Did you want to be more specific in the `when` argument then?

I believe you can specify `when="+raja+cuda ^raja@:0.14"`.

Reply:
Great point, thanks! I have added that to both packages.

This is something that was a soft requirement of the RAJA/Umpire suite for a while but should only apply to specific versions...